examples/usecases/transformers-next-item-prediction-with-pretrained-embeddings.ipynb

```diff
@@ -0,0 +1,5658 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #14.        batch_size=10,
</pre><p>If you run it on the full dataset, batch_size=10 will take a long time?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/1102/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>